### PR TITLE
active_fulfillment 3.x

### PR DESCRIPTION
**Pull request by wvanbergen**
Preparation for a major release.

- Fix namespace to be in line with gem name.
- Update active_utils dependency.
- Clean up tests to use Minitest directly.
- Remove `amazon` fulfillment service

@ntalbott @jnormore @kmcphillips

### CODE REVIEW

**Comment by wvanbergen** [comment]
Tests are passing now. Anything else we would like to tackle for a major release?

**Comment by ntalbott** [comment]
I don't use it... but if there are any existing deprecations, might be a good time to kill 'em?

File: README.md [hunk 1]
```diff
@@ -42,3 +45,4 @@ as well as to make testing more straightforward.
 * Simon Eskildsen
 * Tobias Lütke
 * Tom Burns
+* Willem van Bergen
```
Comment (pickle27) [inline README.md]: haha nice

Comment (pickle27) [inline README.md]: I should add me :p

**Comment by wvanbergen** [comment]
I think I will merge this if everyone agrees. The remote tests are not working yet, but they weren't working before either. I will do a separate PR to fix them. @pickle27 @kmcphillips thoughts?

**Comment by kmcphillips** [comment]
Yeah, I agree with splitting up remote tests.

**Comment by pickle27** [comment]
yup I am okay with that

**Pull request merged**